test/framework/src/main/java/org/opensearch/index/shard/IndexShardTestCase.java

```diff
@@ -622,7 +622,7 @@ protected IndexShard newShard(
         Settings nodeSettings = Settings.builder().put("node.name", routing.currentNodeId()).build();
         DiscoveryNodes discoveryNodes = IndexShardTestUtils.getFakeDiscoveryNodes(routing);
         // To simulate that the node is remote backed
-        if (indexMetadata.getSettings().get(IndexMetadata.SETTING_REMOTE_STORE_ENABLED) == "true") {
+        if (indexMetadata.getSettings().getAsBoolean(IndexMetadata.SETTING_REMOTE_STORE_ENABLED, false)) {
```

Comment:
Just to confirm `false` here is the default value, right?

Reply:
Yup.
https://github.com/opensearch-project/OpenSearch/blob/94987937d77f4aac65e5e57a56a3ab5f45b73bc5/server/src/main/java/org/opensearch/cluster/metadata/IndexMetadata.java#L354-L356